pyclam/tests/test_graph.py

```diff
@@ -80,22 +84,62 @@ def test_clear_cache(self):
         return
 
     def test_bft(self):
-        visited = self.manifold.graph.bft(next(iter(self.manifold.graph.transition_clusters)))
+        visited = self.manifold.graph.bft(next(iter(self.manifold.graph.walkable_clusters)))
         self.assertGreater(len(visited), 0)
         self.assertLessEqual(len(visited), self.manifold.graph.cardinality)
         return
 
     def test_dft(self):
-        visited = self.manifold.graph.dft(next(iter(self.manifold.graph.transition_edges)))
+        visited = self.manifold.graph.dft(next(iter(self.manifold.graph.walkable_edges)))
         self.assertGreater(len(visited), 0)
         self.assertLessEqual(len(visited), self.manifold.graph.cardinality)
         return
 
     def test_random_walks(self):
         results = self.manifold.graph.random_walks(
-            starts=list(self.manifold.graph.transition_clusters),
+            starts=list(self.manifold.graph.walkable_clusters),
             steps=100,
         )
         self.assertGreater(len(results), 0)
         [self.assertGreater(v, 0) for k, v in results.items()]
         return
+
+    # def test_replace(self):
```

Comment:
I would think `unittest.skip` would be nice here instead of leaving it commented out. Let's whoever runs the code know there's work in flight